basket/news/tasks.py

```diff
@@ -32,6 +32,10 @@
                                iso_format_unix_timestamp, parse_newsletters, parse_newsletters_csv,
                                SUBSCRIBE, UNSUBSCRIBE)
 
+# datetime.datetime is not a submodule, rather datetime is a class
+# within the datetime module.
+# See https://stackoverflow.com/questions/27491472/how-to-directly-import-now-from-datetime-datetime-submodule
+now = datetime.datetime.now
```

Comment:
OOOO right. My apologies. This works fine though. Alternatively you can just `from datetime import datetime` and then use `datetime.now()` and `datetime.fromtimestamp()` around the file. Up to you really.